src/core/credentials/transport/tls/spiffe_utils.cc

```diff
@@ -18,16 +18,28 @@
 
 #include "src/core/credentials/transport/tls/spiffe_utils.h"
 
+#include <openssl/x509.h>
+
 #include <string>
 
 #include "absl/strings/match.h"
 #include "absl/strings/str_cat.h"
 #include "absl/strings/str_format.h"
 #include "absl/strings/str_split.h"
+#include "src/core/tsi/ssl_transport_security_utils.h"
+#include "src/core/util/json/json_object_loader.h"
+#include "src/core/util/json/json_reader.h"
+#include "src/core/util/load_file.h"
 #include "src/core/util/status_helper.h"
 
 namespace grpc_core {
 namespace {
+constexpr absl::string_view kAllowedUse = "x509-svid";
+constexpr absl::string_view kAllowedKty = "RSA";
+constexpr int kx5cSize = 1;
+constexpr absl::string_view kCertificatePrefix =
+    "-----BEGIN CERTIFICATE-----\n";
+constexpr absl::string_view kCertificateSuffix = "\n-----END CERTIFICATE-----";
 
 constexpr absl::string_view kSpiffePrefix = "spiffe://";
 constexpr int kMaxTrustDomainLength = 255;
```

Comment:
Please group all of the constants together and alphabetize

Reply:
Done